homework.md

```diff
@@ -4,77 +4,80 @@
 
 What is 2 + 2?
 
-5
+4
 
 ## Question 2
 
 What is the opposite of Black?
 
-Purple
+White
 
 ## Question 3
 
-What is the  2 * 3 * 6 * 1 * 10 * 34?
+What is the 2 _ 3 _ 6 _ 1 _ 10 \* 34?
 
-1322
+12240
 
-## Question 4 
+## Question 4
 
 What does HTML stand for?
 
-Hyper Tall Massive Language
+Hypertext Markup Language
 
 ## Question 5
 
 What is CSS useful for?
 
-Making a website go faster
+Making a website look more presentable by adding colours, layout and fonts.
 
 ## Question 6
 
 Using CSS, how would I select every `<h1>` tag on the page?
 
 ```css
-#h1 {
-
+h1 {
 }
 ```
 
 ## Question 7
 
 What is JavaScript?
 
-A programming language used on the internet
+A programming language used on the website to make it interactive
 
 ## Question 8
 
 What three problem does Git & Github solve?
 
 When people want to show off code to each other they can put it on Github
 
+Can help to edit someone else codes or make projects
+
+Can keep track of different versions of a file
+
 ## Question 9
 
 What happens when you `fork` a repository?
 
-You delete it
+You make a copy of the original repository on your GitHub account
 
-## Question 10 
+## Question 10
 
 What happens when you clone a repostory?
 
-It send it to a friend
+The repository is copied to your local machine with the help of Git.
 
 ## Question 11
 
 What is a Pull Request?
 
-When you send a file over the internet
+Asks the repository owner to merge the changes with the original repository
 
```

Comment:
There can also be a case where you are the repository owner and you want to send a Pull Request to your team members. 

@Hozan94 in the case I just described, what is the purpose of the pull request?

Reply:
@louisechow So in this case, the purpose of "Pull Request" is to tell collaborators that a change has been `pushed` (or merged ?) to the original repository.